packages/orbit-components/src/Illustration/Illustration.ct.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+import { test, expect } from "@playwright/experimental-ct-react";
+
+import { NAMES } from "./consts";
+import IllustrationStory from "./Illustration.ct-story";
+
+test.describe("visual Illustration", () => {
+  NAMES.map(name =>
+    test(`Illustration ${name}`, async ({ mount }) => {
```

Comment:
Do we need all these screenshots for these components? Wouldn't be enough to have one screenshot with all the Illustrations?

Reply:
It might be, but I feel it will be easier to identify which illustration is broken, when they are separated

Reply:
Yep, but if it's a 1px difference on a big screenshot -> not comfortable to look for where exactly it is :D

Reply:
But no strong opinion on this, can be both ways, as you suggested or separated

Reply:
No really strong opinion either. I've just got overwhelmed by the number of screenshots 😅 

Let's ask what the team thinks.